README.md

```diff
@@ -23,20 +25,51 @@ Building Tempest requires that you have the following available on your Linux sy
 - `sleep`
 - `tar`
 
-The (work-in-progress) build tools will (eventually) download the following dependencies:
+### Via Build Tool
+
+The build tool will eventually take care of the entire process of installing the remaining dependencies and building Tempest. However it is in progress, so it will only take care of the first part of the build process.
+
+Run `make toolchain` to install the following dependencies (using the in-progress build tool):
+
+- [Go](https://go.dev/)
+- [TinyGo](https://tinygo.org/)
+
+For now, you will need to manually copy the newly installed Go into a usual location. (Once the build tool is complete, this step will not be necessary)
+
+```
+sudo cp -r ./toolchain/go-* /usr/local/go
+PATH="/usr/local/go/bin:$PATH"
+go version # confirm that go is installed
```

Comment:
If the idea is to have a local version of go, what about modifying `$PATH` and `$GOPATH` in the current session to point to the local toolchain, instead of copying this exe to the global location?

Reply:
Oh right, obviously that's the way to go. Though I'm not sure I need to change `GOPATH`, I assumed it would be pointing at `~/go`. I'll try it.